packages/@stylexjs/babel-plugin/__tests__/transform-process-test.js

```diff
@@ -9,37 +9,64 @@
 
 jest.autoMockOff();
 
+import path from 'path';
 import { transformSync } from '@babel/core';
 import stylexPlugin from '../src/index';
 
-function transform(source, opts = {}) {
-  const { code, metadata } = transformSync(source, {
-    filename: opts.filename || '/stylex/packages/test.stylex.js',
-    parserOpts: {
-      flow: 'all',
+function transformWithConsts(source, opts = {}) {
+  const fixturesDir = path.join(__dirname, '__fixtures__');
+
+  const pluginOpts = {
+    ...opts,
+    unstable_moduleResolution: {
+      rootDir: fixturesDir,
+      type: 'commonJS',
+      resolveFilePath(importPath) {
+        if (importPath === './constants.stylex') {
+          return [
+            'themeNameRef',
+            path.join(fixturesDir, 'constants.stylex.js'),
+          ];
+        }
+        return null;
+      },
     },
-    plugins: [
-      [
-        stylexPlugin,
-        {
-          unstable_moduleResolution: {
-            rootDir: '/stylex/packages/',
-            type: 'commonJS',
-          },
-          ...opts,
-        },
-      ],
-    ],
+  };
+
+  const main = transformSync(source, {
+    filename: path.join(fixturesDir, 'main.stylex.js'),
+    parserOpts: { sourceType: 'module' },
+    babelrc: false,
+    plugins: [[stylexPlugin, pluginOpts]],
   });
-  return { code, metadata };
+
+  const consts = transformSync(
+    `
+    import * as stylex from '@stylexjs/stylex';
+    export const constants = stylex.defineConsts({
+      YELLOW: 'yellow',
+      ORANGE: 'var(--orange)',
+      mediaBig: '@media (max-width: 1000px)',
+      mediaSmall: '@media (max-width: 500px)'
+    });
+    `,
+    {
+      filename: path.join(fixturesDir, 'constants.stylex.js'),
+      parserOpts: { sourceType: 'module' },
+      babelrc: false,
+      plugins: [[stylexPlugin, pluginOpts]],
+    },
+  );
+
+  return stylexPlugin.processStylexRules(
+    [...(main.metadata.stylex || []), ...(consts.metadata.stylex || [])],
+    opts.useLayers,
+  );
```

Comment:
I found it confusing that just in this test the return value of the "transform" function is actually the CSS generated by the process function.